Applications/spire/source/charting/chart_view.cpp

```diff
@@ -216,14 +216,9 @@ ChartValue ChartView::calculate_step(ChartValue::Type value_type,
 
 QString ChartView::get_axis_string(ChartValue::Type type,
     ChartValue value) const {
-  if(type == ChartValue::Type::DURATION) {
-    std::ostringstream ss;
-    ss << static_cast<time_duration>(value);
-    return QString::fromStdString(ss.str());
-  } else if(type == ChartValue::Type::MONEY) {
+  if(type == ChartValue::Type::DURATION || type == ChartValue::Type::MONEY ||
+      type == ChartValue::Type::QUANTITY) {
     return m_item_delegate->displayText(to_variant(type, value), QLocale());
-  } else if(type == ChartValue::Type::QUANTITY) {
-    return QString::number(static_cast<double>(static_cast<Quantity>(value)));
```

Comment:
Use the delegate for the label too.

Reply:
Is there a way to get a truncated Money or Quantity from it?

Reply:
The delegate performs truncation.

Reply:
But how does it know how many decimals to return? I need 3 decimal places for the label and 2 for the y-axis values.

By default it's giving me 5+ (the rest are off screen).

Reply:
Good point, use the item_delegate for now and what we'll have to do is add a feature to the item delegate about how many decimal places to display. That will come later.